src/main/java/formflow/library/data/SubmissionRepositoryService.java

```diff
@@ -85,9 +88,18 @@ public Submission findOrCreate(HttpSession httpSession) {
     var id = (UUID) httpSession.getAttribute("id");
     if (id != null) {
       Optional<Submission> submissionOptional = findById(id);
-      return submissionOptional.orElseGet(Submission::new);
+      if (submissionOptional.isEmpty()) {
```

Comment:
If we are going to add log messages, are there other ones that would add value as well?  

Maybe here: https://github.com/codeforamerica/form-flow/blob/main/src/main/java/formflow/library/FileController.java#L90. (this is the latest code, we'd have to find the spot in the 0.0.7 code).

Reply:
I think our log from `findOrCreate` would also cover the `FilController`.